contracts/TokenConverter/AbstractTokenConverter.sol

```diff
@@ -980,12 +1019,25 @@ abstract contract AbstractTokenConverter is AccessControlledV8, IAbstractTokenCo
     /// @param amountOut Amount of tokenAddressOut transferred from this converter
     function _preTransferHook(address tokenOutAddress, uint256 amountOut) internal virtual {}
 
+    /// @dev Checks if amount to convert is greater than minimum amount to convert or not
+    /// @param amountIn The amount to convert
+    /// @param tokenAddress Address of the token
+    /// @return isValid true if amount to convert is greater than minimum amount to convert
+    function _validateMinAmountToConvert(uint256 amountIn, address tokenAddress) internal returns (bool isValid) {
+        priceOracle.updateAssetPrice(tokenAddress);
+        Exp memory baseAssetPrice = Exp({ mantissa: priceOracle.getPrice(tokenAddress) });
+        uint256 amountInMinInUsd = mul_ScalarTruncate(baseAssetPrice, amountIn);
+
+        if (amountInMinInUsd >= minAmountToConvert) {
```

Comment:
Why `Min` in the name of the variable. I think it could generate some confusion:

```suggestion
        uint256 amountInInUsd = mul_ScalarTruncate(baseAssetPrice, amountIn);

        if (amountInInUsd >= minAmountToConvert) {
```

Reply:
Done https://github.com/VenusProtocol/protocol-reserve/pull/48/commits/554d804df86900e232a5055e1ca3c85a9f0d169e